CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 # Release History
 
 ## 1.1.0-beta.1 (Unreleased)
+### New Features
+- Public preview version of a new set of APIs to simplify the experience using IoT Plug and Play:
+  - az_iot_pnp_client_init
+  - az_iot_pnp_client_get_user_name
+  - az_iot_pnp_client_get_client_id
+  -	az_iot_pnp_client_sas_get_signature
+  -	az_iot_pnp_client_sas_get_password
+  -	az_iot_pnp_client_telemetry_get_publish_topic
+  -	az_iot_pnp_client_commands_parse_received_topic
+  -	az_iot_pnp_client_commands_response_get_publish_topic
+  -	az_iot_pnp_client_property_parse_received_topic
+  -	az_iot_pnp_client_property_document_get_publish_topic
+  -	az_iot_pnp_client_property_patch_get_publish_topic
```

Comment:
i feel like tabs are being used here. the rest of this doc uses spaces so just double check this uses spaces?

Reply:
done. had to open this on notepad to find the tabs...

Reply:
you can also use VSCode and turn on boundary whitespaces. makes it pretty easy to find.
![image](https://user-images.githubusercontent.com/36710865/98260379-1ec83900-1f38-11eb-97a2-a4a9aee28480.png)